manifest_prod.yml

```diff
@@ -6,10 +6,25 @@ env:
   NEW_RELIC_CONFIG_FILE: newrelic.ini
   NEW_RELIC_ENV: production
 applications:
-- name: web
+- name: web-a
   path: .
   buildpack: "https://github.com/ddollar/heroku-buildpack-multi.git"
   host: fec-prod-web
+  services:
+    - fec-creds-prod
+  env:
+    NEW_RELIC_APP_NAME: OpenFEC Web (production)
+    FEC_WEB_API_URL: https://api.open.fec.gov/
+    FEC_FORCE_HTTPS: true
+    # These must be set manually via `cf set-env`
```

Comment:
Would it make sense to note somewhere (or just right here) all the additional varnames we now need to make sure are manually set on the creds service?

```
# FEC_WEB_API_KEY
# FEC_WEB_API_KEY_PUBLIC
# NEW_RELIC_LICENSE_KEY
```

... And any others I might have missed.